crates/zk-por-core/src/account.rs

```diff
@@ -30,6 +30,14 @@ impl Account {
         hash
     }
 
+    pub fn get_empty_account(num_of_tokens: usize) -> Account {
```

Comment:
in my PR27, I also have a function to compute empty accounts and empty proofs. Pls uniformly use my functions. Otherwise, your computed root hash will diverge from my prover computed.